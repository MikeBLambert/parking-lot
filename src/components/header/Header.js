import React from 'react';
import styles from './Header.css';

export default function Header() {
  return (
    <div id={styles.header}>
      <h1>Herb's Parking Lot</h1>
      <nav>
        <a>Search Cars</a>
        <a>Search Lots</a>
        <a>Create Cars</a>
      </nav>
    </div>
  );
}
