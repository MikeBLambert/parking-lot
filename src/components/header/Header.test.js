import React from 'react';
import Header from './Header';
import { shallow } from 'enzyme';

describe('Header component', () => {

  it('renders a Header component', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
  });

  it('Has a title and a nav with three links', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper.html()).toContain('<h1>');
    expect(wrapper.html()).toContain('<nav>')
    expect(wrapper.find('a')).toHaveLength(3);
  });
});
