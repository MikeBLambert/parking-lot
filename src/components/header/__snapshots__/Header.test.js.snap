// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders a Header component 1`] = `
<div
  id="header"
>
  <h1>
    Lot Keeper
  </h1>
  <nav>
    <a
      href="/cars/search"
    >
      Search Cars
    </a>
    <a
      href="/"
    >
      Search Lots
    </a>
    <a
      href="/cars/register"
    >
      Register a Car
    </a>
  </nav>
</div>
`;
